cmd/check.go

```diff
@@ -138,7 +140,6 @@ func runCheckCmd(cmd *cobra.Command, args []string) {
 	}
 
 	// pull out useful properties
-	ctx := initData.ctx
 	workspace := initData.workspace
```

Comment:
an reason not to use the ctx var to tidy things up?

Reply:
tidied up